app/Http/Controllers/GridController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/*
+* Status: Public
+* Description: Grid Template
```

Comment:
Thoughts on a more descriptive description? Maybe `Grid Promotion Template` to say it's coming from promotions? Trying to think from the perspective of someone selecting it from the dropdown in the CMS.